src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main-content">
+      <div class="calendar calendar--start-day-thu calendar--month-length-30">
```

Comment:
Checklist item 2
https://github.com/mate-academy/layout_calendar/blob/master/checklist.md
```suggestion
      <div class="calendar calendar--start-day--thu calendar--month-length--30">
```